### PR TITLE
Removed y offset on the APV structure as it wasn't useful anymore.

### PR DESCRIPTION
**Pull request by JulienIgnace-Unity**
### Purpose of this PR
Removed the Y-offset code from the APV.
As we currently allocate the maximum space for the index buffer this does not save memory anymore and just costs complexity and performance.

---
### Testing status
Tested with the APV test scene repo.

### CODE REVIEW

**Review by sebastienlagarde — APPROVED** [review]

File: com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeBrickIndex.cs [hunk 1]
```diff
@@ -75,7 +69,7 @@ struct BrickMeta
         internal ProbeBrickIndex(Vector3Int indexDimensions)
         {
             Profiler.BeginSample("Create ProbeBrickIndex");
-            int index_size = indexDimensions.x * (indexDimensions.y + 1) * indexDimensions.z;
+            int index_size = indexDimensions.x * indexDimensions.y * indexDimensions.z;
```
Comment (FrancescoC-unity) [inline com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeBrickIndex.cs:72]: Need to also update same calculation in InitProbeReferenceVolume (removing the +1)

Comment (FrancescoC-unity) [reply com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeBrickIndex.cs:72]: Also same in Clear here (can't link for whatever reason, but in the diff is line 125 of old and 111 of new.

File: com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeVolume.hlsl [hunk 2]
```diff
@@ -134,18 +132,14 @@ bool TryToGetPoolUVWAndSubdiv(APVResources apvRes, float3 posWS, float3 normalWS
     bool hasValidUVW = true;
 
     // transform into APV space
-    float3 posRS = mul(_WStoRS, float4(posWS + normalWS * _NormalBias
-                                             + viewDirWS * _ViewBias, 1.0)).xyz;
+    float3 posRS = mul(_WStoRS, float4(posWS + normalWS * _NormalBias + viewDirWS * _ViewBias, 1.0)).xyz;
```
Comment (FrancescoC-unity) [inline com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeVolume.hlsl:135]: Why changing this? It is a long line and to me in a single line makes it a bit harder to parse

**Review by alelievr — APPROVED** [review]

**Comment by FrancescoC-unity** [comment]
I am trying it with APV_Test project, simple scene and it seems to fail with args out of range

**Comment by JulienIgnace-Unity** [comment]
@FrancescoC-unity haven't tried again after latest changes. Let me take a look.

**Pull request merged**